main.go

```diff
@@ -58,7 +60,11 @@ var (
 				"https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml",
 			},
 			Knative: []string{
-				"https://storage.googleapis.com/knative-releases/build/latest/release.yaml",
+				// NOTE: build release 0.4.0 is currently broken, using nightly instead
+				// https://github.com/knative/build/issues/566
+				"https://storage.googleapis.com/knative-nightly/build/latest/build.yaml",
```

Comment:
the "latest" manifest is inherently untested, let's go back to knative-releases so when a release is cut we don't need to do anything. Even if the latest release is broken, this manifest is doing its job.